doc/quickstart.md

```diff
@@ -0,0 +1,202 @@
+Welcome to
+# Pyccel
+
+ ## What is Pyccel?
+
+- static compiler for Python 3, using Fortran or C as backend language.
+- started as small open-source project in 2018 at IPP Garching.
+- public repository is now hosted on GitHub, freely available for download.
+
+## Python’s objects, variables, and garbage collection
+
+ Python is an **interpreter** language, **dynamically typed** and **garbage-collected**.
```

Comment:
Typo `interpreter`->`interpreted`

interpreter is the object, interpreted is an adjective